src/Microsoft.DotNet.Wpf/src/Shared/cpp/Utils.cxx

```diff
@@ -5,10 +5,6 @@
 #include "Utils.hxx"
 #include <shlwapi.h>
 
-#if _MANAGED
```

Comment:
A copy of this file is used by PresentationNative (i.e., not managed :-)). 

Since this is under Shared, it gets packaged by `Microsoft.DotNet.Arcade.Wpf.Sdk` transport-package, and flows into the private `dotnet-wpf-int` repository, where it is consumed by the build for PresentationNative_cor3.dll. 

https://github.com/dotnet/wpf/blob/55815c9287682c1621cd5f89ffdd75ae770f4ea6/packaging/Microsoft.DotNet.Arcade.Wpf.Sdk/Microsoft.DotNet.Arcade.Wpf.Sdk.ArchNeutral.csproj#L38-L41



In short, you can't do this.

Reply:
I'm afraid I don't quite understand. What I'm removing has no effect when the file is _not_ managed, as the `#if` would evaluate to false then. Furthermore, all that these blocks contain is an unused `using namespace System.Security`. This using statement is meaningless when not compiling managed, and on top of that it isn't even used. Uses of the types inside this namespace were removed in e4a212b4fd7d3bb6bce7d88cf6f0471a9fc8e6f4, but the using statement (and the empty `#if` block below) were not removed at the same time.

Reply:
You're right - I wasn't looking too carefully. I was still thinking this had the old system::security CAS declarations. You can definitely do this. I'll build your branch against the internal branch just to be sure everything builds ok and let you know.